__mocks__/react-i18next.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+/* eslint-disable react/prop-types */
+import { translate, translateWithFormatting } from '@/utils/test/i18n';
+import React, { FunctionComponent } from 'react';
+
+const Trans: FunctionComponent<{ i18nKey: string }> = ({
```

Comment:
Can this component be called `Translate` instead

Reply:
I can do for the local const, but the export will need to be called `Trans` as that's the `react-i18next` api